### PR TITLE
docs(vrl): add casing functions (https://github.com/vectordotdev/vrl/…

### PR DESCRIPTION
**Pull request by leshow**
[docs for vrl PR](https://github.com/vectordotdev/vrl/pull/973)

### CODE REVIEW

**Comment by leshow** [comment]
Not quite sure how to make the function names not set off the spellcheck?

**Review by jszwedko — APPROVED** [review]
Great, thanks @leshow ! We'll merge this once VRL has been upgraded in Vector as part of the next Vector release.

**Comment by jszwedko** [comment]
I added the case names as allowed words for the spell checker in [b54f8c9](https://github.com/vectordotdev/vector/pull/21021/commits/b54f8c937db9a6f7d0beacaca248f9f223aeb3a1)

File: website/cue/reference/remap/functions/camelcase.cue [hunk 1]
```diff
@@ -0,0 +1,43 @@
+package metadata
+
+remap: functions: camelcase: {
+	category: "String"
+	description: """
+		Takes the `value` string, and turns it into camelCase. Optionally, you can
+		pass in the existing case of the function, or else we will try to figure out the case automatically.
```
Comment (buraizu) [inline website/cue/reference/remap/functions/camelcase.cue:7]:
```suggestion
		pass in the existing case of the function, or else an attempt is made to determine the case automatically.
```

**Review by buraizu — APPROVED** [review]
Thanks, approving with a minor phrasing update requested

**Pull request merged**